index2.html

```diff
@@ -29,6 +29,7 @@
               Home</a></li>
           <li class="is-active"><a href="#" aria-current="page">
               Joke of the Day</a></li>
+          <li> <input type = "button" onclick="joke()" value = "Joke Me"></input></li>
```

Comment:
look at me I am amie (not really) 
shouldn't this be attached to the button in the joke area.